store/tikv/2pc.go

```diff
@@ -455,11 +458,11 @@ func (c *twoPhaseCommitter) commitSingleBatch(bo *Backoffer, batch batchKeys) er
 	if regionErr != nil {
 		err = bo.Backoff(BoRegionMiss, errors.New(regionErr.String()))
 		if err != nil {
-			return errors.Trace(err)
+			return errors.Trace(errWrapper(err))
 		}
 		// re-split keys and commit again.
 		err = c.commitKeys(bo, batch.keys)
-		return errors.Trace(err)
+		return errors.Trace(errWrapper(err))
```

Comment:
`commitKeys` will call `commitSingleBatch` which should have wrapped the error already.